src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,122 @@
 package baseball;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+
+        Scanner sc = new Scanner(System.in);
+
+        // 컴퓨터 수 생성
+        int comNum = getComputerNumber();
+        int[] comDigits = Stream.of(String.valueOf(comNum).split(""))
+                .mapToInt(Integer::parseInt)
+                .toArray();
+
+        System.out.println("숫자 야구 게임을 시작합니다.");
+
+        playNumBaseballGame(sc, comDigits);
```

Comment:
scanner 객체를 통째로 보내는 것 보다 number 변수를 만들어서 보내는 건 어떨까요?
```suggestion
        String playerDigits = ~
        playNumBaseballGame(playerDigits, comDigits);
```